tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
```

Comment:
What about ClientError?
If it's by design I'd add the "raise ClientError" in the comment of the public function.

Reply:
It's by design. I've added it to the docstring.